src/main/java/edu/hillel/springmvclesson29/controller/Orders.java

```diff
@@ -0,0 +1,55 @@
+package edu.hillel.springmvclesson29.controller;
+
+import edu.hillel.springmvclesson29.model.Order;
+import edu.hillel.springmvclesson29.response.ResponseBodyInterface;
+import edu.hillel.springmvclesson29.services.OrderRepository;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+import java.util.Objects;
+
+@RestController
+@RequestMapping("/orders")
+public class Orders {
```

Comment:
Please use OrdersController instead of just Orders

Reply:
Well done.